### ISSUE TITLE
Problema no acesso ao banco de dados

### ISSUE THREAD

**Issue by Anders1232** [bionimbuz/Bionimbuz#7]
Seguindo os passos, do readme, o primeiro passo da configuração do banco MySQL é insuficiente, pois não é explicado comop cadastrar um usuário no banco de dados, ao tentar usar o usuário do sistema e até mesmo o root ocorre o erro 1698 (28000): access denied for user.

**Comment by Brenokx** [comment]
Não sei como está hoje, mas o cadastro do usuário era feito pela interface, e o usuário root era adicionado assim que subia a aplicação core, https://github.com/bionimbuz/Bionimbuz/blob/master/src/main/java/br/unb/cic/bionimbuz/BioNimbuZ.java#L130

**Comment by Anders1232** [comment]
Então após rodar o prepare.sh basta ir em bin/src/main/java/br/unb/cic/bionimbuz/ e executar BioNimbuZ.class?

**Comment by jefferson-chaves-gomes** [comment]
Não... Ainda é necessário seguir até o passo 3 do README (Configurações), ou seja, é necessário:

1. Criar um banco de dados chamado bionimbuz_db;
2. Configurar o acesso a sua base de dados;
3. Informar ao Bionimbuz os dados de acesso para a sua base de dados;
4. Configurar o conf/credentials.yaml

Após isto, vc pode iniciar a aplicação.

**Comment by Anders1232** [comment]
Pequena observação: a linha de configuração do mysql(bind-address) estava no arquivo "/etc/mysql/mariadb.conf.d/50-server.cnf"
